drizzle-zod/src/index.ts

```diff
@@ -60,6 +60,7 @@ import {
 } from 'drizzle-orm/pg-core';
 import {
 	SQLiteBlobJson,
+  SQLiteBoolean,
```

Comment:
Small nitpick, 
indentation is off 🙃

Reply:
@raducristianpopa we use dprint for code formatting, please format the code with it.